packages/openapi-generator/src/parser/unique-naming.spec.ts

```diff
@@ -2,32 +2,104 @@ import { pascalCase } from '@sap-cloud-sdk/util';
 import { ensureUniqueNames } from './unique-naming';
 
 describe('ensureUniqueNames', () => {
-  it('replaces duplicate names using defaults', () => {
-    expect(ensureUniqueNames(['duplicate', 'duplicate'])).toEqual([
-      'duplicate',
-      'duplicate1'
-    ]);
-  });
+  describe('with strictNaming disabled', () => {
+    const options = { strictNaming: true };
```

Comment:
I know the meaning of the option, but `strictNaming disabled` => `{ strictNaming: true }` seems a bit confusing.

Reply:
A simple way might be `with strictNaming true`.

Reply:
Oh, the text is is just wrong...